eddies/trello-webhook
# 2020 03 dependencies update

**Check for required environment variables and update dependencies**
* * *

# What does this Pull Request do?
If the environment variables `TRELLO_API_KEY` or `TRELLO_TOKEN` are not available, log it (console.error).

Updates:
- @google-cloud/functions-framework from 1.3.2 to 1.5.0
- eslint from 6.5.1 to 6.8.0
- eslint-config-airbnb-base from 14.0.0 to 14.1.0
- eslint-plugin-import from 2.18.2 to 2.20.1
- jest from 24.9.0 to 25.1.0
- nodemon from 1.19.3 to 2.0.2

# How should this be tested?
unset either (or both) `TRELLO_API_KEY` or `TRELLO_TOKEN`, e.g.:
```console
unset TRELLO_API_KEY
```
then
```console
npm test
```
which should cause tests to fail. 

Set both variables (any arbitrary string), e.g.:
```console
export TRELLO_API_KEY=abc
export TRELLO_TOKEN=xyz
```

and now `npm test` should pass. 


# Checklist
